llvm/lib/Target/RISCV/RISCVRegisterInfo.h

```diff
@@ -110,13 +110,31 @@ struct RISCVRegisterInfo : public RISCVGenRegisterInfo {
     return RC;
   }
 
-  bool doesRegClassHavePseudoInitUndef(
-      const TargetRegisterClass *RC) const override {
+  bool isRVVRegClass(const TargetRegisterClass *RC) const {
```

Comment:
Nit, `isRVVRegClass` sounds like it should include the segmented reg classes since they're a part of RVV too.  Maybe name it something like `isVRRegClass` and `isVRNRegClass`?